homeassistant/components/pyload/switch.py

```diff
@@ -18,6 +18,7 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import PyLoadConfigEntry
+from .coordinator import pyLoadData
```

Comment:
Why is this a lowercase "p"?
If it's a class, it should start with an uppercase "P".